tracing-undertow/src/test/java/com/palantir/tracing/undertow/TracedRequestHandlerTest.java

```diff
@@ -70,7 +70,8 @@ public void before() {
 
         traceReportedLatch = new CountDownLatch(1);
         port = portSelector.incrementAndGet();
-        HttpHandler nextHandler = new TracedRequestHandler(ResponseCodeHandler.HANDLE_200);
+        HttpHandler nextHandler = new TracedRequestHandler(exchange -> exchange.getResponseHeaders()
+                .put(HttpString.tryFromString("requestId"), exchange.getAttachment(TracingAttachments.REQUEST_ID)));
```

Comment:
should we make this `_requestId` for consistency and to avoid potential clobbering